mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -103,9 +103,14 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 	METHOD method_39458 contains (DDD)Z
 		ARG 1 x
 		ARG 3 z
+		ARG 5 offset
```

Comment:
`offset` doesn't look right. This parameter is used to expand the box in all directions.